yt/odin/checks/bin/unaware_nodes/__main__.py

```diff
@@ -1,9 +1,26 @@
 from yt_odin_checks.lib.check_runner import main
 
+DEFAULT_ENABLED_NODE_FLAVORS = [
+    "data",
+    "exec",
+    "tablet",
+    "chaos"
+]
 
-def find_unaware_nodes(yt_client):
+
+def find_unaware_nodes(yt_client, check_node_flavors):
```

Comment:
`check_node_flavors` – sounds like a boolean, maybe call it `node_flavors_to_check`? (here and in options)